src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -13,179 +13,306 @@ Sending all your application errors to Sentry ensures you'll be notified in real
 
-Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, delete and discard it.
+When the next minute begins, we again record up to 60 events, dropping the rest until the minute is up. It continues like this for the remainder of the hour. After an hour, we re-calculate your per-minute limit, and for the next hour use that limit to decide whether events get dropped each minute. We repeat this process every hour until Sentry eventually accepts all events.
 
-### Resolve
+If, instead of a single big spike (or an overall, permanent, increase in traffic), you experience many small spikes, there may be many days in a row when at least a few events are dropped. In that scenario, spike protection remains active the entire time.
 
-You've been alerted on a new error in your code? Go to the **[Issue Details](/product/issues/issue-details/)** page to get all the data you need to know about the error. If it's a real issue in your code, assign a team member to resolve it. Don't forget to let Sentry know once it's resolved. Learn more about workflows for handling issues in [Issue States and Triage](/product/issues/states-triage/).
+Spike protection is an organization-level setting, so once it's triggered, it affects all the projects in the organization, regardless of which project triggered the spike.
 
-### Delete & Discard
+### Managing a Spike {#-spike-protection-was-activated----what-should-i-do}
+
+If spike protection has been triggered for your account, you'll receive an email notifying you:
+
+![Spike Protection Email](spike-email.jpg)
+
+To manage the current spike and avoid future spikes, we recommend taking the following steps:
+
+- Set up an [on-demand budget](#on-demand-budget) to ensure you have time to adjust your volume in the event of a future spike in errors.
+- Set better [rate limits](#6-rate-limiting) on the DSN keys for the projects related to the spike.
+- If it's a specific release version that has caused the spike, add the version identifier to the project's [inbound filters](#3-inbound-data-filters) to avoid accepting events from that release.
+
+To review the error events dropped because of spike protection, go to the "Usage Stats" tab of **Stats** for your Sentry org and select "Errors" in the "Category" dropdown.
+
+### When Does Spike Protection Become "Inactive?"
+
+Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ It does however mean that the next time events are dropped, with respect to alerts it will count as a "reactivation" of spike protection.
```

Comment:
This last sentence sounds like a penalty of some sort; is it? Or is just a fact?

Reply:
You could rephrase this to say that the next time spike protection is activated you will be alerted. There are a lot of statements in the negative in this block which makes the content hard to follow:

> Events will be dropped during any minute in which you send more than the hourly limit that Sentry has calculated for you. 

Feels easier to follow.